src/presentational-components/shared/table-toolbar-view.js

```diff
@@ -113,6 +121,18 @@ export const TableToolbarView = ({
         { ...rows.length > 0 && { actionResolver } }
         areActionsDisabled={ areActionsDisabled }
         rowWrapper={ rowWrapper }
+        sortBy={ sortByState }
+        onSort={ (e, index, direction) => {
+          const key = columns[index - isSelectable].key;
```

Comment:
No need for this, it can be simplified
```JS
setSortByState({ index, direction });
fetchData({
            ...pagination,
            offset: 0,
            name: '',
            orderBy: `${direction === 'desc' ? '-' : ''}${columns[index - isSelectable].key}`
          });
```